docs/Configuration.md

```diff
@@ -80,6 +80,19 @@ The host name of the machine on which the MySQL server is running.
 
 If this property is unspecified, the value of [DB.database_host] is used instead.
 
+If this property is set to `localhost` (but not `127.0.0.1` nor `::1`),
+then the [MYSQL.port] property is discarded as the driver connects using
```

Comment:
```
then the value of the [MYSQL.port] property is discarded as the driver connects using
```

Reply:
I think the text should be moved under "### port" instead.

Reply:
Ok. I'll change to keep this message only in the MYSQL.port section.